frontend/packages/citizen-frontend/src/localization/sv.tsx

```diff
@@ -172,11 +172,12 @@ const sv: Translations = {
                   Du kan ansöka om plats i småbarnspedagogisk verksamhet året
                   om. Ansökningen bör lämnas in senast fyra månader före behovet
                   av verksamheten börjar. Om behovet börjar med kortare varsel
-                  bör du ansöka om plats senast två veckor före.
+                  pga. sysselsättning eller studier bör du ansöka om plats
+                  senast två veckor före.
                 </P>
                 <P>
                   Du får ett skriftligt beslut om platsen. Beslutet delges i
-                  tjänsten{' '}
```

Comment:
Just to be on the safe side: there will be whitespace rendered between "tjänsten" and anchor element?